tox.ini

```
@@ -94,8 +95,8 @@ basepython = python3.11
 
 # always generate with the min supported versions
 deps =
-    Django>=3.2,<3.3
-    wagtail>=5.2,<6.0
+    Django>=4.2,<5.0
+    wagtail>=5.2,<6.3
```

```suggestion
    wagtail>=5.2,<6.0
```

as we're generating migrations with the minimum supported versions as per comment